api/src/services/tus/server.ts

```diff
@@ -90,6 +92,20 @@ export async function createTusServer(context: Context): Promise<[Server, () =>
 				});
 			}
 
+			const eventMeta = {
+				payload: await service.readOne(file.id),
+				key: file.id,
+				collection: 'directus_files',
+			};
+
+			const eventContext = {
+				database: getDatabase(),
+				schema: req.schema,
+				accountability: req.accountability,
+			};
+
+			emitter.emitAction('files.upload', eventMeta, eventContext);
```

Comment:
As we are not mutating or re-using the meta and context variables should we inline them?

Reply:
Sure, looked nicer when doing the whole `readOne` but without that there is no upside